### PR TITLE
Documentation improvemation

### PR DESCRIPTION
**Pull request by lucasferreiralimax**
* Reflection information in legacy, api, installation, homepage.
* Reduce SVG
* Addition alt in images
* Style in images for max-width and resize page

@kazupon i believe this good for upgrade, code review please.

### CODE REVIEW

File: vuepress/README.md [hunk 1]
```diff
@@ -60,6 +60,14 @@ footer: MIT Licensed | Copyright © 2020 kazuya kawaguchi
...
 </div>
 
+<div class="patreon" style="padding: 0 0 24px 0;">
+  <p style="text-align: center;">
+    <a href="https://www.patreon.com/kazupon" target="_blank">
+      <img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patreon">
+    </a>
+  </p>
+</div>
+
```
Comment (kazupon) [inline vuepress/README.md:70]: I use GitHub Sponsors as my main patron services, so please remove the patreon. 🙏

Comment (lucasferreiralimax) [reply vuepress/README.md:70]: Hi @kazupon, but remove all occurrences of the patreon?

File: vuepress/zh/guide/pluralization.md [hunk 2]
```diff
@@ -126,8 +128,8 @@ const messages = {
   }
 }
 ```
-Where the format is `0 things | things count ends with 1 | things count ends with 2-4 | things count ends with 5-9, 0 and teens (10-19)`.
-P.S. Slavic pluralization is a pain, you can read more about it [here](http://www.russianlessons.net/lessons/lesson11_main.php).
+格式在哪里 `0 东西 | 事情以结尾结束 1 | 事情以结尾结束 2-4 | 事情以结尾结束 5-9, 0 和青少年 (10-19)`.
+附言 斯拉夫多元化是困难的，您可以阅读有关它的更多信息 [这里](http://www.russianlessons.net/lessons/lesson11_main.php).
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/pluralization.md:132]: @kazupon review this translate to chinese, and i rewrite word "pain" to "difficult" ok?

File: vuepress/zh/guide/number.md [hunk 3]
```diff
@@ -55,3 +58,76 @@ new Vue({
...
 </div>
 ```
+
+## 自定义格式
+
+:::tip 支持版本
+:new: 8.10+ 新增
+:::
+
+`$n` 方法返回的结果字符串带有完全格式化的数字，该数字只能作为整体使用。 在需要格式化格式化数字的某些部分（例如小数位）的情况下，`$n` 是不够的。 在这种情况下，`<i18n-n>` 功能组件将有所帮助。
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/number.md:68]: @kazupon review this translate to chinese

File: vuepress/zh/guide/messages.md [hunk 4]
```diff
@@ -102,7 +107,57 @@ const messages = {
 ```html
 <p>DIO: the world !!!!</p>
 ```
+### 格式化链接的语言环境消息
 
+如果语言区分字符大小写，则可能需要控制链接的语言环境消息的大小写。
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/messages.md:112]: @kazupon review this translate for chinese

File: vuepress/zh/guide/locale.md [hunk 5]
```diff
@@ -46,3 +48,8 @@ export default {
 :::warning 警告
 :warning: 对于使用了 `sync: false` 的组件，语言环境的更改将被忽略。
 :::
+
+:::warning 组件与根范围
+:warning: 在组件内更改 `$i18n.locale` 不会更新根语言环境。
+如果您依靠根语言环境，例如在使用 [root fallbacks](./fallback.html) 时，请使用 `$root.$i18n.locale` 而不是$ `i18n.locale`。
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/locale.md:54]: @kazupon review this translate for chinese

File: vuepress/zh/guide/lazy-loading.md [hunk 6]
```diff
@@ -45,18 +45,25 @@ function setI18nLanguage (lang) {
...
-  if (i18n.locale !== lang) {
-    if (!loadedLanguages.includes(lang)) {
-      return import(/* webpackChunkName: "lang-[request]" */ `@/lang/${lang}`).then(msgs => {
-        i18n.setLocaleMessage(lang, msgs.default)
-        loadedLanguages.push(lang)
-        return setI18nLanguage(lang)
-      })
-    }
+export function loadLanguageAsync(lang) {
+  // 如果语言相同
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/lazy-loading.md:49]: @kazupon review this translate for chinese

File: vuepress/zh/guide/hot-reload.md [hunk 7]
```diff
@@ -1,11 +1,23 @@
 # 热重载
 
+您可以使用Webpack的 [Hot Module Replacement](https://webpack.js.org/concepts/hot-module-replacement/) (HMR) 功能来监视本地化文件中的更改以及将热更改重新加载到您的应用程序中。
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/hot-reload.md:3]: @kazupon review this translate to chinese

File: vuepress/zh/guide/fallback.md [hunk 8]
```diff
@@ -1,5 +1,20 @@
 # 回退本地化
 
+*总结：使用`fallbackLocale：'<lang>'`选择首选语言缺少翻译时要使用的语言。*
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/fallback.md:3]: @kazupon review this translate for chinese

File: vuepress/zh/guide/component.md [hunk 9]
```diff
@@ -84,18 +84,67 @@ new Vue({
 
 如果你希望在组件语言环境中进行本地化，可以在 `i18n` 选项中用 `sync: false` 和 `locale`。
 
+## 组件的共享语言环境消息
```
Comment (lucasferreiralimax) [inline vuepress/zh/guide/component.md:87]: @kazupon  review this translate for chinese

File: vuepress/README.md (hunk 1 above)
Comment (kazupon) [reply vuepress/README.md]: @lucasferreiralimax 
Hmm, if you can remove them, i hope it. 🙏

**Review by lucasferreiralimax — COMMENTED** [review]
Hello @kazupon i addition some reviews for information in docs chinese please review all translate.

In style scoped i addition more examples with ::v-deep and /deep/

File: vuepress/README.md (hunk 1 above)
Comment (lucasferreiralimax) [reply vuepress/README.md]: Yes, i remove in this [commit](https://github.com/kazupon/vue-i18n/pull/1104/commits/d94b193deb8c7307bcfba5a96fa8cf139f853cef) review please

**Comment by kazupon** [comment]
@lucasferreiralimax 
Thank you for your imorovemention!

There are a few review points about Chinese, but I think they are OK, so I would like to merge them once.
If there are any problems, I think that vue-i18n docs get feedback in the PR.

**Comment by lucasferreiralimax** [comment]
> @lucasferreiralimax
> Thank you for your imorovemention!
> 
> There are a few review points about Chinese, but I think they are OK, so I would like to merge them once.
> If there are any problems, I think that vue-i18n docs get feedback in the PR.

Very good 👍🏽
I agree with you

**Review by kazupon — APPROVED** [review]

**Pull request merged**